contracts/Org.sol

```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: BSD 3-Clause
 
 pragma solidity ^0.6.10;
+pragma experimental ABIEncoderV2;
```

Comment:
What is this for?

Reply:
ABIEncoderV2 is an optional pragma that enables arbitrary encoding/decoding of structs. This is required for the events where we emit a struct, e.g. `emit GrantFinalized(grant);` and `emit ClaimCreated(newClaim);`. So without this pragma, you'd have to, for example, individually emit each field of the struct as a separate event parameter.

You can check out the docs on ABIEncoderV2 [here](https://solidity.readthedocs.io/en/v0.6.12/layout-of-source-files.html#experimental-pragma). Even though it's called an "experimental" pragma, it is no longer consider experimental (they just kept the pragma the same) and is safe to use.

Reply:
Sg